tests/nlu/classifiers/test_diet_classifier.py

```diff
@@ -307,6 +311,16 @@ async def test_train_persist_load_with_different_settings(
     create_diet(config, load=True, finetune=True)
 
 
+@pytest.mark.timeout(240, func_only=True)
+async def test_train_persist_load_with_nested_dict_config(
+    create_train_load_and_process_diet: Callable[..., Message],
+    create_diet: Callable[..., DIETClassifier],
+):
+    config = {HIDDEN_LAYERS_SIZES: {"text": [256, 512]}}
```

Comment:
Detail: you can bring the test time of this down by specifying `ENTITY_RECOGNITION: False` (because TF upgrade slowed DIET specifically with entity recognition down)